libs/expo/shared/ui-components/src/lib/Picker/Picker.ios.tsx

```diff
@@ -0,0 +1,118 @@
+import { ChevronLeftIcon } from '@monorepo/expo/shared/icons';
+import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import { Picker as RNPicker } from '@react-native-picker/picker';
+import { useState } from 'react';
+import { Pressable, StyleSheet, View } from 'react-native';
+import Modal from 'react-native-modal';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import TextBold from '../TextBold';
+import TextRegular from '../TextRegular';
+
+interface IPickerProps {
```

Comment:
maybe import from `./Picker` as we'd want a uniform interface for Picker.
It can always be extended here if needed.